x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/benchmark.ts

```diff
@@ -0,0 +1,223 @@
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('GET /internal/cloud_security_posture/benchmark', () => {
+    let agentPolicyId: string;
+    let agentPolicyId2: string;
+    let agentPolicyId3: string;
+    let agentPolicyId4: string;
+
+    beforeEach(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+      await esArchiver.load('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+
+      const { body: agentPolicyResponse } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy',
+          namespace: 'default',
+        });
+
+      agentPolicyId = agentPolicyResponse.item.id;
+
+      const { body: agentPolicyResponse2 } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy 2',
+          namespace: 'default',
+        });
+
+      agentPolicyId2 = agentPolicyResponse2.item.id;
+
+      const { body: agentPolicyResponse3 } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy 3',
+          namespace: 'default',
+        });
+
+      agentPolicyId3 = agentPolicyResponse3.item.id;
+
+      const { body: agentPolicyResponse4 } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy 4',
+          namespace: 'default',
+        });
+
+      agentPolicyId4 = agentPolicyResponse4.item.id;
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId,
+        'cspm',
+        'cloudbeat/cis_aws',
+        'aws',
+        'cspm',
+        'CSPM-1'
+      );
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId2,
+        'kspm',
+        'cloudbeat/cis_k8s',
+        'vanilla',
+        'kspm',
+        'KSPM-1'
+      );
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId3,
+        'vuln_mgmt',
+        'cloudbeat/vuln_mgmt_aws',
+        'aws',
+        'vuln_mgmt',
+        'CNVM-1'
+      );
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId4,
+        'kspm',
+        'cloudbeat/cis_k8s',
+        'vanilla',
+        'kspm',
+        'KSPM-2'
+      );
+    });
+
+    afterEach(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+      await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+    });
+
+    it(`Should return non-empty array filled with Rules if user has CSP integrations`, async () => {
+      const { body: res }: { body: GetBenchmarkResponse } = await supertest
+        .get(`/internal/cloud_security_posture/benchmarks`)
+        .set(ELASTIC_HTTP_VERSION_HEADER, '1')
+        .set('x-elastic-internal-origin', 'xxx')
```

Comment:
we can also import it from  http-common as we do for `ELASTIC_HTTP_VERSION_HEADER`

```js
import { X_ELASTIC_INTERNAL_ORIGIN_REQUEST } from '@kbn/core-http-common';
```

Reply:
makes sense, updated. I also was thinking about making this less repetitive to add default headers, but not sure if it's worth the time.